tw-tasks-core-spring-boot-starter/src/main/java/com/transferwise/tasks/core/autoconfigure/TwTasksCoreAutoConfiguration.java

```diff
@@ -251,4 +253,9 @@ public JavaxValidationEnvironmentValidator twTasksJavaxValidationEnvironmentVali
   public IPartitionKeyStrategy twTasksPartitionKeyStrategy() {
     return new RandomPartitionKeyStrategy();
   }
+
+  @Bean
+  public NoOpTaskHandler noOpActionTaskHandler(TasksProperties tasksProperties) {
```

Comment:
Let's add @ConditionalOnMissingBean annotation, so a service can extend that class. E.g. add logging.

Also let's use `twTasks` perfix, so the chance of bean names conflicts in a service is smaller.

Reply:
Oh I see, this is why all beans had that in this class! Will add, and change the prefix!